home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/invitations/InvitationApiIT.java

```diff
@@ -0,0 +1,96 @@
+package com.softserveinc.ita.homeproject.api.tests.invitations;
+
+import com.softserveinc.ita.homeproject.ApiResponse;
+import com.softserveinc.ita.homeproject.api.CooperationApi;
+import com.softserveinc.ita.homeproject.api.tests.utils.ApiClientUtil;
+import com.softserveinc.ita.homeproject.api.tests.utils.MailHogUtil.ApiMailHogUtil;
+import com.softserveinc.ita.homeproject.api.tests.utils.MailHogUtil.Dto.ResponseDto;
+import com.softserveinc.ita.homeproject.model.*;
+import org.apache.commons.lang3.RandomStringUtils;
+import org.junit.jupiter.api.Test;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class InvitationApiIT {
+
+    private final CooperationApi cooperationApi = new CooperationApi(ApiClientUtil.getClient());
+
+    @Test
+    void isEmailSentTest() throws Exception {
+        CreateCooperation createCoop = createCooperation();
+        ApiResponse<ReadCooperation> cooperationWithHttpInfo = cooperationApi.createCooperationWithHttpInfo(createCoop);
+
+        TimeUnit.MILLISECONDS.sleep(60000);
+
+        ResponseDto response = ApiMailHogUtil.getMessages();
+
+        assertTrue(getLastMessageEmailTo(response).contains(createCoop.getAdminEmail()));
+        assertTrue(response.getCount() > 0);
+        assertTrue(getLastMessageSubject(response).contains("invitation to cooperation"));
+    }
+
+    private String getLastMessageEmailTo(ResponseDto responseDto){
+        return String.valueOf(responseDto.getItems().get(0).getContent().getHeaders().getTo());
+    }
+
+    private String getLastMessageSubject(ResponseDto responseDto){
+        return String.valueOf(responseDto.getItems().get(0).getContent().getHeaders().getSubject());
+    }
+
+    private CreateCooperation createCooperation() {
+        return new CreateCooperation()
+                .name("newCooperationTest")
+                .usreo(RandomStringUtils.randomAlphabetic(10))
+                .iban(RandomStringUtils.randomAlphabetic(20))
+                .adminEmail("adminemail@gmail.com")
```

Comment:
tell again, use correct email (test.receive.messages@gmail.com), because try to send message to not correct mail our test mail will be block

Reply:
@likeRewca what do you mean? This mail for Mailhog, it doesn't matter which one email, because anyway this test won't pass if mailhog is not running

Reply:
although, even if the test fails, the message will still be sent, so it makes sense to change, if our test email could be blocked

Reply:
Ok, mb I don`t understand that email for mailhog.